apps/air-discount-scheme/backend/src/app/modules/flight/flight.controller.ts

```diff
@@ -47,10 +54,77 @@ import { HttpRequest } from '../../app.types'
 export class PublicFlightController {
   constructor(
     private readonly flightService: FlightService,
+    @Inject(forwardRef(() => DiscountService))
```

Comment:
ditto with the injection

Reply:
This was proving difficult to circumvent